tests/bsim/bluetooth/audio/src/cap_acceptor_test.c

```diff
@@ -645,6 +630,7 @@ static int set_supported_contexts(void)
 void test_start_adv(void)
 {
 	int err;
+	struct bt_le_ext_adv *ext_adv;
```

Comment:
I guess we never stop or delete the advertising set since it could be moved here?

Reply:
Correct. At some point in time we may want to change that behaviour, but I'll leave it as is for now